crates/uv-cli/src/lib.rs

```diff
@@ -1709,6 +1709,10 @@ pub struct PipShowArgs {
 #[derive(Args)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct PipTreeArgs {
+    /// Show the version constraint(s) imposed on each package.
+    #[arg(long)]
+    pub show_version_specifiers: bool,
```

Comment:
I had to remove this from `uv tree`. It actually doesn't work right now because we don't write the specifiers to the lockfile. We only write the resolved versions. (So `--show-version-specifiers` was just showing `==` for every annotation.)